linkedevents/settings.py

```diff
@@ -135,11 +135,11 @@ def sentry_anonymize_user_repr(obj, hint):
     STATIC_URL=(str, "/static/"),
     SUPPORT_EMAIL=(str, ""),
     SYSTEM_DATA_SOURCE_ID=(str, "system"),
-    TOKEN_AUTH_ACCEPTED_AUDIENCE=(str, "https://api.hel.fi/auth/linkedevents"),
+    TOKEN_AUTH_ACCEPTED_AUDIENCE=(list, "https://api.hel.fi/auth/linkedevents"),
     TOKEN_AUTH_ACCEPTED_SCOPE_PREFIX=(str, "linkedevents"),
-    TOKEN_AUTH_AUTHSERVER_URL=(str, "https://api.hel.fi/sso/openid"),
-    TOKEN_AUTH_FIELD_FOR_CONSENTS=(str, "https://api.hel.fi/auth"),
-    TOKEN_AUTH_REQUIRE_SCOPE_PREFIX=(bool, True),
+    TOKEN_AUTH_AUTHSERVER_URL=(list, "https://api.hel.fi/sso/openid"),
+    TOKEN_AUTH_FIELD_FOR_CONSENTS=(list, "https://api.hel.fi/auth"),
```

Comment:
Default values should be lists... I think? 🤔 not sure if environ automatically casts strings into lists or not.

Reply:
true true